File: assignment 3/power.cpp
#include<iostream>
using namespace std;
class Power
{
    int base,num=1,exp,i;
    public : void display()
    {
        cout<<"enter the base number:\n";
        cin>>base;
        cout<<"enter the exponential number:\n";
        cin>>exp;
        for(i=1;i<=exp;i++)
        {
            num=num*base;

        }
        cout<<base<<"^"<<exp<<"="<<num;


    }
};
int main()
{
    Power p1;
    p1.display();
}

File: JAVAAssignment 3/Power.java
import java.util.Scanner;
class Power
{
public static void main(String[] args)
{
int base,num=1,exp,i;
System.out.println("Enter the base number");
Scanner sc=new Scanner(System.in);
base=sc.nextInt();
System.out.println("Enter the exponential number");
exp=sc.nextInt();
for(i=1;i<=exp;i++)
{
num=num*base;
}
System.out.println(num);
}
}